scripts/build.py

```diff
@@ -174,47 +181,127 @@ def _ensure_files_exist(filepaths):
         filepaths: list(str). Paths to files that we want to ensure exist.
 
     Raises:
-        Exception: Raised if one or more of the files does not exist.
+        OSError: One or more of the files does not exist.
     """
     for filepath in filepaths:
         if not os.path.isfile(filepath):
-            raise Exception('File %s does not exist.' % filepath)
+            raise OSError('File %s does not exist.' % filepath)
 
 
-def process_html(source_path, target_path, file_hashes):
-    """Copies contents of HTML file, while removing whitespace and
-    replacing paths inside the file with paths with hashes.
+def get_file_count(directory_path):
+    """Count total number of file in directory, ignoring any files with
```

Comment:
Count the total number of files in the given directory, ...

Reply:
Done!